release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -219,6 +219,11 @@ You can now configure the time-to-live (TTL) duration of both successful and uns
 
 For more information, see xref:../networking/dns-operator.adoc#nw-dns-cache-tuning_dns-operator[Tuning the CoreDNS cache].
 
+[id=ocp-4-12-egress-ip-multitenant]
+==== Egress IP support for multiple tenants on {rh-openstack-first}
+
+{rh-openstack}, paired with {product-title}, now uses an egress IP address for network traffic from many tenants in a cluster.
```

Comment:
I guess better to mention that EgressIP support  for both OVNKubernetes and OpenshiftSDN.

Reply:
@huiran0826 This information will be in the docs portion for sure, however for the release notes the summary statement gives the reader an overview of the feature. Let me know if you have additional input. Thanks!

Reply:
ok

Reply:
Can we change this to:
~~~
Egress IP support on {rh-openstack-first}
~~~

The multiple tenants has nothing to do with the change itself

Can we also change the sentence to:
~~~
{rh-openstack}, paired with {product-title}, now supports automatic attachment and detachment of EgressIP addreses so that the traffic from one or more pods in one or more namespaces has a consistent source IP address for services outside the cluster network.
~~~